### DISCUSSION TITLE
Test cluster running k3s. Will envoy gateway work or are do I need a full k8 cluster?

### DISCUSSION THREAD

**Discussion by robd003** [envoyproxy/gateway#4909] [Q&A]
I'm setting up a test cluster to just run basic checks during our CI builds of our platform. Can I use k3s with envoy gateway or will it not work without additional dependencies?

**Comment by arkodg** [comment]
should work if k3s supports `LoadBalancer` type services

**Reply by renanramonh** [reply]
It works I run it in my personal k3s cluster.

**Reply by robd003** [reply]
Awesome, thanks for the confirmation @renanramonh

**Reply by robd003** [reply]
@renanramonh did you have to make any special changes in the configuration or did envoy gateway just "work out of the box" like in k8s?

**Reply by renanramonh** [reply]
It just works like in any other k8s cluster, I did not have to change anything just for k3s.
